app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html

```diff
@@ -81,6 +82,21 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_export.arriving.h3")</h
         <li><a href="@{appConfig.checkDelaysInlandBorderFacilityUrl}">@messages("inspection_needed_export.arriving.p1.link2")</a></li>
     </ul>
 
+    @govukInsetText(InsetText(
```

Comment:
is this change needed only for export but not for import ?

Reply:
I was about to say the same: the ticket says EU to GB as well.

Reply:
Don the import as well.